src/constants/oefenBestand.js

```diff
@@ -0,0 +1,63 @@
+import {inventory} from "./inventory.js";
+
+// Opdracht 1a: Gebruik een array-methode om een array te maken met daarin alle tv-type namen. Log de uitkomst in de console.
+
+const tvType = inventory.map((inventory) => {
+    return inventory.type;
+});
+
+console.log(tvType);
+
+//     Opdracht 1b: Gebruik een array-methode om alle informatie te verzamelen van de tv's die volledig uitverkocht zijn. Log de uitkomst in de console.
+
+const soldOutTvs = inventory.filter((tv) => {
+    if (tv.originalStock === 0) {
```

Comment:
```suggestion
    if (tv.originalStock === tv.sold) {
```
Anders krijg je tv's terug waarvoor nooit voorraad voor is ingekocht